.expeditor/config.yml

```diff
@@ -157,7 +157,7 @@ subscriptions:
           post_commit: true
       - trigger_pipeline:deploy/automate-chef-io:
           post_commit: true
-      - bash:.expeditor/purge-cdn.sh:
+      - purge_packages_chef_io_fastly:{{channel}}/automate/latest:
```

Comment:
`buildkite_hab_build_group_published` does not have a `channel` metadata (https://expeditor.chef.io/docs/reference/workloads/#buildkite_hab_build_group_published). You'll need to specify the channel (which I think is `dev`?)